README.md

```diff
@@ -112,7 +118,7 @@ Then finally build it by running these commands:
 	cmake .. -G "MSYS Makefiles"
 	make
 
-Please note that this build of mGBA for Windows is not suitable for distribution, due to the scattering of DLLs it needs to run, but is perfect for development.
+Please note that this build of mGBA for Windows is not suitable for distribution, due to the scattering of DLLs it needs to run, but is perfect for development. However, if you *really* want to distribute your mGBA build (e.g. for testing on machines that don't have the MSYS2 environment installed), you can use a tool called "[Dependency Walker](http://dependencywalker.com)" to see which additional DLL files you need to ship with your mGBA binary. 
```

Comment:
How about "However, if distributing such a build is desired" instead? Better tone.

Please remove the space at the end of the line.